common/resources/src/main/res/values/strings.xml

```diff
@@ -101,4 +101,55 @@
+    <string name="main_consoleRpc">Console RPC</string>
+    <string name="main_consoleRpc_details">Kizzy allows you to use some predefined presets to set your presence on Discord.</string>
+    <string name="main_experimentalRpc">Experimental RPC</string>
+    <string name="main_experimentalRpc_details">Experimental RPC is a feature in Kizzy that constantly switches between Apps RPC and Media RPC.</string>
+    <string name="main_customRpc_details">Kizzy allows you to create your own fully customisable rich presence for Discord.</string>
+    <string name="main_comingSoon">Coming Soon</string>
+    <string name="exit">Exit</string>
+    <string name="logs">Logs</string>
+    <string name="drawer_subtitle_help">HELP</string>
+    <string name="drawer_faq">FAQ</string>
+    <string name="go_to_profile">Go to Profile</string>
+    <string name="version">Version</string>
+    <string name="try_again">Retry</string>
+    <string name="request_for_notification_access">Notification Access is needed for app to extract media information.</string>
+    <string name="update_check_for_update">Checking for update...</string>
+    <string name="update_no_updates_available">No updates available</string>
+    <string name="compact_mode">Compact Mode</string>
+    <string name="are_you_sure">Are you sure?</string>
+    <string name="yes">Yes</string>
+    <string name="user_profile_error">Could not Update User Profile:</string>
+    <string name="no_internet_user_profile">No Internet connection. Try again later.</string>
+    <string name="user_profile_rpc_name">USING KIZZY RICH PRESENCE</string>
+    <string name="user_profile_bio">ABOUT ME</string>
+    <string name="user_profile_special_button">Special Button</string>
+    <string name="user_profile">User Profile</string>
+    <string name="logout">Logout</string>
+    <string name="custom_rpc_directory">Directory to store Custom RPC configs</string>
+    <string name="select_directory">Select directory</string>
+    <string name="enter_details">Enter details</string>
+    <string name="status_online">Online</string>
+    <string name="status_idle">Idle</string>
+    <string name="status_dnd">Do Not Disturb</string>
+    <string name="status_invisible_offline">Invisible and shown as offline</string>
+    <string name="status_offline">Offline</string>
+    <string name="notification_access">Notification Access Permission</string>
+    <string name="post_notification">Post Notification</string>
+    <string name="request_for_post_notification">Grant Permission to Show Notification</string>
+    <string name="storage_access">Storage Access</string>
+    <string name="start_app_now">Start App Now</string>
+    <string name="skip">Skip</string>
+    <string name="party_current">Party Current Size</string>
+    <string name="party_max">Party Max Size</string>
+    <string name="user_profile_party">(%1$d of %2$d)</string>
+    <string name="auto_scroll">Auto scroll</string>
```

Comment:
Some contributors wont know if there are new strings to be translated so do you mind adding all these with default values(english) in all other strings file.